packages/mocha-reporter/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@realm/mocha-reporter",
+  "version": "0.0.1",
+  "description": "A custom reporter for mocha",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc ",
+    "prepare": "rm -rf dist && npm run build",
+    "start": "tsc -w"
+  },
+  "files": [
+    "dist"
+  ],
+  "author": {
+    "name": "Realm",
+    "email": "help@realm.io",
+    "url": "https://realm.io"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/realm/realm-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/realm/realm-js/issues"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "dependencies": {
+    "mocha": "9.1.1"
```

`mocha-remote` has a peer dependency on `mocha` in version 8. It's probably best to use the same version here.

```suggestion
    "mocha": "^8"
```